ci/upload_app_distribution_mainnet.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
upload script remains basically the same. We can leave just one upload_app_distribution.sh file and just pass arguments to this bash script

Reply:
changed in 0d9a4935498e880bf0da1640e17e5203deb0bb58